docs/UserGuide.adoc

```diff
@@ -20,172 +20,228 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+****
 
-=== Editing a person : `edit`
+Format 2: `add [-h] {title} on {date} at {time}` +
+Example: `add project on friday at 2359`
+****
+* Supports keywords _on_, _at_ and _days_
+****
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+=== Deleting a task : `delete / del / -d / -`
 
+Format 1: `delete [-h] {index}` +
+Example: `delete 1`
 ****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the last person listing. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
+* Deletes task at that index (positive number) in the most recent call of list
 ****
 
-Examples:
+Format 2: `delete [-h]  {title} on {date} at {time}` +
+Example: `delete CS2103 Project V0.1`
+****
+* Deletes task that has same name as the given
+* Prompts user to choose according to date and time
+****
 
-* `edit 1 p/91234567 e/johndoe@example.com` +
-Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-* `edit 2 n/Betsy Crower t/` +
-Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+=== Listing all tasks : `list / -l`
 
-=== Finding all persons containing any keyword in their name: `find`
+Format: `list [-h]` +
+Example: `list`
+****
+* Lists title and deadlines of all uncompleted tasks
+****
 
-Finds persons whose names contain any of the given keywords. +
-Format: `find KEYWORD [MORE_KEYWORDS]`
+=== Finding a task : `find / -f`
 
+Format: `find [-h] {keyword}` +
+Example: `find work`
 ****
-* The search is case insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans` will match `Hans Bo`
+* Scans through all tasks to find for keyword in any attribute and list it
 ****
 
-Examples:
+=== Undoing the last action : `undo / -u`
 
-* `find John` +
-Returns `John Doe` but not `john`
-* `find Betsy Tim John` +
-Returns Any person having names `Betsy`, `Tim`, or `John`
-
-=== Deleting a person : `delete`
+Format: `undo` +
+Example: `undo`
+****
+* Undos the last successful action executed
+****
 
-Deletes the specified person from the address book. Irreversible. +
-Format: `delete INDEX`
+=== Editing a task : `edit / -e`
 
+Format: `edit [-h] {index} m/[new main] d/[new date] n/[new note] tag/[new tags] a/[alarm] r/[recurrence] u/[urgent]` +
+Example: `edit 2 m/CS2103`
```

Comment:
Better to also provide an example of the command with all params (most complex).